libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/incident/IncidentInfo.kt

```diff
@@ -52,6 +53,7 @@ class IncidentInfo internal constructor(
     val longDescription: String?,
```

Comment:
Line 49: seriously? We have a property called alertcCodes?..

Reply:
yes, what's wrong with it?

Reply:
yep, because it is  alert-c codes